.github/workflows/merge-to-main-copy.yml

```diff
@@ -0,0 +1,20 @@
+# Branch main-copy is a copy of main to allow for deployment to a separate environment
```

Comment:
Yes, that makes sense. The reason why I picked a 'weird' name (`main-copy`) is to make clear that we shouldn't work with this branch directly (i.e. someone could checkout `dev` and commit code to it). Also, a name such as `dev` or `test` could suggest that it contains code that's under development. But the purpose of the branch is to show to others how the app fully works (with mock data because the real data isn't sufficient for all functionality).

What do you think? I'm open to suggestions :)

Reply:
That makes sense. On a 'dev' branch I would indeed expect new development. So let's keep it 'main-copy' then.